testBaseData/test_database_config.yaml

```diff
@@ -1,5 +1,20 @@
 schema:
   instanceName: sars_cov-2_minimal_test_config
+  nucleotideSequences:
+    - name: main
+  aminoAcidSequences:
+    - name: E
+    - name: M
+    - name: N
+    - name: S
+    - name: ORF1A
+    - name: ORF1B
+    - name: ORF3A
+    - name: ORF6
+    - name: ORF7A
+    - name: ORF7B
+    - name: ORF8
+    - name: ORF9B
```

Comment:
Why do we need to configure that here? Isn't this information already clear from reading the `reference-genomes.json`?

Reply:
Right, this was from the first proposal and was supposed to be deleted here